scopes.md

```diff
@@ -0,0 +1,147 @@
+
+### conversations
+
+This scope gives access to private messaging, i.e. the following requests:
+- Create conversations
+- Send private messages
+
+### email
+
+This scope is defined by the OpenID Connect spec. It allows reading user's own email address using the UserInfo route or from the ID Token.
+
+### interactions
+
+This scope gives access to the following requests:
+- Create/delete comments
+- Report posts/comments
+- Post/remove likes
+- Subscribe/mute posts
+- Hide posts from streams
+- Voting in polls
+
+### notifications
+
+This scope gives access to reading notifications.
+
+### openid
+
+This scope is defined by the OpenID Connect spec. This is a mandatory scope. It allows using OpenID Connect authentication.
+
+### private:read
+
+This scope gives access to the following requests:
+- Read private posts
+- Read private posts interactions (likes, comments)
+- Reading streams: main stream, aspects stream, mention stream, activity stream, liked stream, commented stream
+
+Note: the `private:read` scope depends on `contacts:read` and can only be granted when `contacts:read` is granted too.
+
+### private:modify
+
+This scope gives access to the following requests:
+- Create private posts
+- Post photos privately
+- Delete private posts
+- Delete private photos
+
+Note: the `private:modify` scope depends on `contacts:read` and can only be granted when `contacts:read` is granted too.
+
+### public:read
+
+This scope gives access to the following requests:
+- Read public profiles of users
+- Perform search among publicly searchable profiles
+- Read public posts
+- Read reshares
+- Read public posts interactions (likes, comments)
+
+Note: The `public:read` scope is granted to any authorized entity even when not explicitly requested.
+
+### public:modify
+
+This scope gives access to the following requests:
+- Create public posts/reshares
+- Post photos
+- Delete public posts
+- Delete public photos
+
+### profile
+
+This scope is defined by the OpenID Connect spec. It allows reading user claims (user properties). Claims are read using UserInfo route or from the ID Token.
+
+User claims as defined by the OpenID Connect are somewhat similar to a users own profile data. Generally this scope means "Allow reading a user's own profile". Besides user claims defined by the standard this scope also gives access to the "read own profile" diaspora* API endpoint.
+
+The `profile` scope gives read access to the following claims supported by diaspora*:
+
+| claim      | meaning                                             |
+| ---------- | --------------------------------------------------- |
+| name       | Full name                                           |
+| nickname   | The left part of diaspora handle before `@` sign    |
+| profile    | User's profile URL address                          |
+| picture    | User avatar URL address                             |
+| gender     | Gender profile field                                |
```

Comment:
I think we need a separate scope for private profiles.

Applications may need access to your profile information to show them in their app, like a username or the profile picture. But most applications won't need access to your gender or the birthdate, so I don't think we should include those unless specifically requested. So private profile fields should be a separate scope, like `private_profile`.